src/notation/inotationconfiguration.h

```diff
@@ -192,6 +192,9 @@ class INotationConfiguration : MODULE_EXPORT_INTERFACE
     virtual muse::ValCh<int> pianoKeyboardNumberOfKeys() const = 0;
     virtual void setPianoKeyboardNumberOfKeys(int number) = 0;
 
+    virtual muse::ValCh<bool> pianoKeyboardNotatedPitch() const = 0;
+    virtual void setPianoKeyboardNotatedPitch(bool state) = 0;
```

Comment:
Let's adjust the name so that it's clear from the name that it's a boolean toggle:

```suggestion
    virtual muse::ValCh<bool> pianoKeyboardUseNotatedPitch() const = 0;
    virtual void setPianoKeyboardUseNotatedPitch(bool state) = 0;
```